source/math/FloatPoint2DVectorList.ooc

```diff
@@ -87,6 +87,54 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
 		}
 		result
 	}
+	offset: func (offset: FloatPoint2D) -> This {
```

Comment:
This function name is a bit ambiguous but from looking at the signature it should be clear what it does, so it's OK I guess.